manifests/worker.pp

```diff
@@ -171,4 +179,12 @@
     ensure => $docker_ensure,
     dns    => $advertise_addr,
   }
+
+  if $xylem_backend {
```

Comment:
I think `if $var {...}` is cleaner for this kind of thing. Are we already using `if $var != undef {...}` elsewhere?